### PR TITLE
Added Game CatchTheBall

### LINKED ISSUE #1609: Title: Catch The Ball Game
### Is your feature request related to a problem? Please describe.

I want to add a new game. It is called Catch the ball.

### Describe the solution you'd like.

Description:
Catch the Ball is an engaging and addictive arcade game that challenges players to catch a bouncing ball with a paddle. 

Key Features:

1)Intuitive Gameplay:

Simple controls: Move the paddle left or right to catch the ball.
Easy to understand, making it accessible for players of all skill levels.

2) Responsive Design:

Mobile-friendly: Optimized for touch controls on smartphones and tablets.
Resizable canvas: Automatically adjusts to different screen sizes, providing a consistent gaming experience on any device.

3) Endless Challenge:

Increasing difficulty: The ball's speed increases over time, testing players' reflexes and coordination.
Score tracking: Players can see their score in real-time and aim to beat their high score.
Screenshot 2024-06-02 132343

### Describe alternatives you've considered.

Please assign me this issue under Gssoc'24

### Add any other context or screenshots about the feature request here.

_No response_

### PR DESCRIPTION
**Pull request by shruti110503**
Added the game successfully.

### 👨‍💻 Changes proposed and Brief Description
I have added the game as proposed. It is made using html css and javascript. 

### ✅ Check List (Check all the applicable boxes) 

- [ ✓] My code doesn't break any part of the project
- [ ✓] This PR does not contain plagiarized content.
- [✓ ] My Addition/Changes works properly and matches the overall repo pattern.
- [ ✓] The title of my pull request is a short description of the requested changes.
catch-the-ball

### CODE REVIEW

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Add your project in index.html

File: index.html [hunk 1]
```diff
@@ -1382,6 +1382,19 @@ <h4>Maze Game</h4>
...
+  
+<div class="maincard">
+  <div class="card1">
+      <img src="30DaysOfJavaScript\assets\catch-the-ball.png alt="Project Image">
+  </div>
+  <div class="card">
+      <h4>CatchTheBall</h4>
+      <p>Catch the ball on the rolling pad and beat your own highscore!</p>
+      <a href="117 - CatchTheBall"><button class="button">Live</button></a>
+      <a href="http://127.0.0.1:5500/117-CatchTheBall/index.html"><button class="button">Github</button></a>
```
Comment (swapnilsparsh) [inline index.html:1394]: This will not be you localhost url, update this.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (shruti110503) [reply index.html:1394]: @swapnilsparsh can you please help me as to how can i add my own localhost url

Comment (swapnilsparsh) [reply index.html:1394]: @shruti110503 Take reference from the other url which are there in the index.htm and make the required changes

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Please resolve the conflict and still the URL is not right.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Please resolve the conflict and add your project in index.html also

**Comment by swapnilsparsh** [comment]
@shruti110503 Not working on mobile

**Comment by swapnilsparsh** [comment]
@shruti110503 Any updates ?

**Comment by shruti110503** [comment]
> @shruti110503 Any updates ?

please check now

**Review by swapnilsparsh — APPROVED** [review]

**Comment by swapnilsparsh** [comment]
@shruti110503 Thank you for contributing to this repo. Looking forward for more 🚀

**Pull request merged**